_posts/2021/2021-11-10-TdoT.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Tag der offenen Tür 2021
+categories: special veranstaltungen
+image: /images/2021-10-30-TdoT.jpg
+author: Marie, Karl
+---
+
+Am 30.10. öffnete die TU Dortmund ihre Pforten, um beim Tag der offenen Tür allen
+BesucherInnen einen Einblick in das Universitätsleben zu vermitteln.
```

Comment:
I think the better gender form would be "Besucher*innen" oder "Besucher:innen".
In line 12 the same.
And maybe also in line 16 for the helpers. Except for that it is a nice post.